### ISSUE TITLE
Bookmark method not working in Google Chrome

### ISSUE THREAD

**Issue by mgonzalezg9** [apankowski/doc88-downloader#14]
When I paste the minified Javascript to download all the pages the method returns a promise but no download starts.
Please check the attachment.
[Screencast from 17-11-23 09:51:19.webm](https://github.com/apankowski/doc88-downloader/assets/48419793/3a40140d-7f9d-4194-ba87-fa24e78fa82a)

**Comment by mgonzalezg9** [comment]
This seems only be happening in Google Chrome. Firefox works as a charm

**Comment by apankowski** [comment]
@mgonzalezg9 thanks for the report. 

I cannot reproduce it on my setup.

Chrome works like a charm on my end, both on Windows 10 and Linux / Ubuntu 22.04 LTS, on multi-page (10-60) doc88 documents. I tested with Chrome 119.0.6045.159 (Official Build) (64-bit). Tested on 10 documents without any issues.

I see you're doing something non-standard, namely pasting the bookmark code into Console, which it's not the intended usage. Also, what's your software setup? Was it possible that it was intermittent internet connection that made the page preload to fail and the whole script to stall? :thinking:
